pkgs/games/multimc/default.nix

```diff
@@ -1,10 +1,10 @@
-{ stdenv, mkDerivation, fetchFromGitHub, cmake, jdk, zlib, file, makeWrapper, xorg, libpulseaudio, qtbase }:
+{ stdenv, fetchFromGitHub, cmake, jdk, zlib, file, makeWrapper, xorg, libpulseaudio, qtbase }:
 
 let
   libpath = with xorg; stdenv.lib.makeLibraryPath [ libX11 libXext libXcursor libXrandr libXxf86vm libpulseaudio ];
-in mkDerivation rec {
+in stdenv.mkDerivation rec {
```

Comment:
please revert this change here that changes what deriver is used.
This package uses `libsForQt5.callPackage` which brings the `qt5` specific `mkDerivation` and libraries into scope.